_includes/pbc/all/install-features/202403.0/install-the-acl-feature.md

```diff
@@ -0,0 +1,298 @@
+|   AclEntityParentMetadata | object | Created | src/Generated/Shared/Transfer/AclEntityParentMetadataTransfer |
+|   AclEntityParentConnectionMetadata | object | Created | src/Generated/Shared/Transfer/AclEntityParentConnectionMetadataTransfer |
+|   AclEntityMetadataCollection | object | Created | src/Generated/Shared/Transfer/AclEntityMetadataCollectionTransfer |
+|   AclEntityMetadataConfig | object | Created | src/Generated/Shared/Transfer/AclEntityMetadataConfigTransfer |
+|   AclRoleCriteria | object | Created | src/Generated/Shared/Transfer/AclRoleCriteriaTransfer |
+|   GroupCriteria | object | Created | src/Generated/Shared/Transfer/GroupCriteriaTransfer |
+|   Groups | object | Created | src/Generated/Shared/Transfer/GroupsTransfer |
+|   Role | object | Created | src/Generated/Shared/Transfer/RoleTransfer |
+|   Roles | object | Created | src/Generated/Shared/Transfer/RolesTransfer |
+|   Rule | object | Created | src/Generated/Shared/Transfer/RuleTransfer |
+|   Rules | object | Created | src/Generated/Shared/Transfer/Transfer |
+|   User | object | Created | src/Generated/Shared/Transfer/UserTransfer |
+|   NavigationItem | object | Created | src/Generated/Shared/Transfer/NavigationItemTransfer |
+|   NavigationItemCollection | object | Created | src/Generated/Shared/Transfer/NavigationItemCollection |
+|   AclUserHasGroupCollection | object | Created | src/Generated/Shared/Transfer/AclUserHasGroupCollection |
+|   AclUserHasGroup | object | Created | src/Generated/Shared/Transfer/AclUserHasGroup |
+|   AclUserHasGroupCriteria | object | Created | src/Generated/Shared/Transfer/AclUserHasGroupCriteria |
+|   AclUserHasGroupConditions | object | Created | src/Generated/Shared/Transfer/AclUserHasGroupConditions |
+
+{% endinfo_block %}
+
+### 4) Set up behavior
+
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN | DESCRIPTION | PREREQUISITES | NAMESPACE |
+|-|-|-|-|
+| AccessControlEventDispatcherPlugin | Adds a listener to the `\Symfony\Component\HttpKernel\KernelEvents::REQUEST` which checks if the user is allowed to access the current resource. |  | Spryker\Zed\Acl\Communication\Plugin\EventDispatcher |
+| AclNavigationItemCollectionFilterPlugin | Checks if the navigation item can be accessed by the current user. |  | Spryker\Zed\Acl\Communication\Plugin\Navigation |
+| AclInstallerPlugin | Fills the DB  with required ACL data. |  | Spryker\Zed\Acl\Communication\Plugin |
+| GroupPlugin | Provides Acl Groups for User. |  | Spryker\Zed\Acl\Communication\Plugin |
+| AclEntityAclRolePostSavePlugin | Saves `RoleTransfer.aclEntityRules` to database. |  | Spryker\Zed\AclEntity\Communication\Plugin\Acl |
+| AclRulesAclRolesExpanderPlugin | Expands `Roles` transfer object with ACL rules. |  | Spryker\Zed\AclEntity\Communication\Plugin\Acl |
+| AclEntityApplicationPlugin | Enables ACL for the whole Application. |  | Spryker\Zed\AclEntity\Communication\Plugin\Application |
```

Comment:
We have to list the following missed plugins: 
* \Spryker\Zed\AclDataImport\Communication\Plugin\AclGroupDataImportPlugin
* \Spryker\Zed\AclDataImport\Communication\Plugin\AclGroupRoleDataImportPlugin
* \Spryker\Zed\AclDataImport\Communication\Plugin\AclRoleDataImportPlugin

Reply:
I don't think it should be done in scope of this ticket. I just moved this IG to 202403.0 and added only related to the current epic changes. IMO we should create a tech dept for such missing things.

Reply:
We previously agreed with the documentation team that it's better to address gaps during regular work. In reality, we may never have time to tackle them if we prioritize technical debt.